manifest.md

```diff
@@ -42,6 +48,8 @@ Unlike the [image index](image-index.md), which contains information about a set
 
         If the manifest uses a different media type than the above, it MUST comply with [RFC 6838][rfc6838], including the [naming requirements in its section 4.2][rfc6838-s4.2], and MAY be registered with [IANA][iana].
 
+        See [Guidelines for Artifact Usage](#guidelines-for-artifact-usage) for other uses of the `config.mediaType` field.
```

Comment:
Does this sentence serve us well for artifacts given the language above it?

> Manifests concerned with portability SHOULD use one of the above media types.

That advice is for manifests of container images. Manifests of artifacts concerned with portability should use a different config media type now. I think we should move the new sentence up into this one.

How about something like:
>    - **`mediaType`** *string*
> 
>         This [descriptor property](descriptor.md#properties) has additional restrictions for `config`.
>         Implementations MUST support at least the following media types:
> 
>         - [`application/vnd.oci.image.config.v1+json`](config.md)
> 
>         Manifests for container images concerned with portability SHOULD use one of the above media types. Manifests for artifacts concerned with portability SHOULD use `config.mediaType` as described in [Guidelines for Artifact Usage](#guidelines-for-artifact-usage).
> 
>         If the manifest uses a different media type than the above, it MUST comply with [RFC 6838][rfc6838], including the [naming requirements in its section 4.2][rfc6838-s4.2], and MAY be registered with [IANA][iana].

I also wonder if we should drop the requirement to support `application/vnd.oci.image.config.v1+json` to allow a "helm charts only" registry or some other such example, I think that lives in seperate PR.

Reply:
That wording looks good to me. Updated.